csharp/extractor/Semmle.Extraction.CSharp/Entities/Symbol.cs

```diff
@@ -9,12 +9,14 @@
 
 namespace Semmle.Extraction.CSharp.Entities
 {
-    public abstract class CachedSymbol<T> : CachedEntity<T> where T : ISymbol
+    public abstract class CachedSymbol<T> : CachedEntity<T> where T : class, ISymbol
     {
-        public CachedSymbol(Context cx, T init)
+        public CachedSymbol(Context cx, T? init)
+#nullable disable
```

Comment:
there's no `nullable` generic type constraint to add onto the baseclass. And if we add `class`, then we can't pass in tuples.